KoeBook.Core/Contracts/Services/ISoundGenerationSelectorService.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace KoeBook.Core.Contracts.Services;
+
+public interface ISoundGenerationSelectorService
+{
+    /// <summary>
+    /// サウンドモデル・スタイルの一覧
+    /// </summary>
+    public IReadOnlyList<SoundModel> Models { get; }
+
+    public ValueTask InitializeAsync(CancellationToken cancellationToken);
+}
+
+public record SoundModel(string name, string[] styles);
```

Comment:
こちらもReadOnlyListにしましょう
```suggestion
public record SoundModel(string name, IReadOnlyList<string> styles);
```